Lesson11-CSS_Positioning/index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="/assets/index.css">
+</head>
+
+
+
+
+<body>
+
+    <header class="header_container nav"> 
+    <div class="header_logo"> 
+        <img class="nav_logo" src="/assets/images/logo.png" alt="logo">
+    </div>
+    <div class="header_list nav_list">
+        <ul>
+            <li class="header_list--item nav_list--item"><a href="#">Services</a></li>
```

Comment:
It looks like we have the BEM element inside DEM element. Its better to create a new BEM block.
Or if you just need the elements the naming can be next:
nav_list -> nav_list-container
nav_list--item -> nav_list-item
Also, the positioning inside BEM block needs to be encapsulated.
Text me if you have questions about it.
Other parts are good! )